sys/syz-extract/netbsd.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/google/syzkaller/pkg/compiler"
+)
+
+type netbsd struct{}
+
+func (*netbsd) prepare(sourcedir string, build bool, arches []string) error {
+	if sourcedir == "" {
+		return fmt.Errorf("provide path to kernel checkout via -sourcedir flag (or make extract SOURCEDIR)")
+	}
+	if !build {
+		return fmt.Errorf("netbsd requires -build flag")
+	}
+	return nil
+}
+
+func (*netbsd) prepareArch(arch *Arch) error {
+	if err := os.Symlink(filepath.Join(arch.sourceDir, "sys", "arch", "amd64", "include"),
```

Comment:
Is this required for netbsd as well? Or it's just a copy-paste from freebsd?

Reply:
Just copy-paste. :smile: